lib/features/manage_account/presentation/forward/forward_controller.dart

```diff
@@ -105,39 +116,28 @@ class ForwardController extends BaseController {
   }
 
   void deleteRecipients(BuildContext context, String emailAddress) {
-    if (_responsiveUtils.isMobile(context)) {
-      (ConfirmationDialogActionSheetBuilder(context)
-        ..messageText(AppLocalizations.of(context)
-            .messageConfirmationDialogDeleteRecipientForward(emailAddress))
-        ..onCancelAction(AppLocalizations.of(context).cancel, () =>
-            popBack())
-        ..onConfirmAction(AppLocalizations.of(context).delete, () =>
-            _handleDeleteRecipientAction({emailAddress})))
-      .show();
-    } else {
-      showDialog(
-        context: context,
-        barrierColor: AppColor.colorDefaultCupertinoActionSheet,
-        builder: (BuildContext context) =>
-          PointerInterceptor(child: (ConfirmDialogBuilder(_imagePaths)
-            ..title(AppLocalizations.of(context).deleteRecipient)
-            ..content(AppLocalizations.of(context)
-                .messageConfirmationDialogDeleteRecipientForward(emailAddress))
-            ..addIcon(SvgPicture.asset(_imagePaths.icRemoveDialog,
-                fit: BoxFit.fill))
-            ..marginIcon(EdgeInsets.zero)
-            ..colorConfirmButton(AppColor.colorConfirmActionDialog)
-            ..styleTextConfirmButton(const TextStyle(
-                fontSize: 17,
-                fontWeight: FontWeight.w500,
-                color: AppColor.colorActionDeleteConfirmDialog))
-            ..onCloseButtonAction(() => popBack())
-            ..onConfirmButtonAction(AppLocalizations.of(context).delete, () =>
-                _handleDeleteRecipientAction({emailAddress}))
-            ..onCancelButtonAction(AppLocalizations.of(context).cancel, () =>
-                popBack()))
-          .build()));
-    }
+    showConfirmDialogAction(currentContext!,
```

Comment:
Why do we need currentContext and where it come from ?

Reply:
1. You can see the function details to see why we need it.
2. It is the `context` of the current state. `GetX` supports this, you can read more about `GetX`